packages/neutrino-middleware-dev-server/index.js

```diff
@@ -0,0 +1,68 @@
+const { pathOr } = require('ramda');
+const opn = require('opn');
+
+module.exports = (neutrino, options = {}) => {
+  const { config } = neutrino;
+  const server = pathOr({}, ['options', 'server'], neutrino);
+  const protocol = process.env.HTTPS ? 'https' : 'http';
```

Comment:
Let's remove all `process.env` variables from this middleware, and have any presets that use this pass them manually.